src/bin/bobc.rs

```diff
@@ -50,7 +50,7 @@ fn main() {
     wait_for_input();
     let get_req = client
         .get(Request::new(GetRequest {
-            key: None,
+            key: Some(BlobKey { key: 0 }),
```

Comment:
You can use enum or tuple struct
```rust
struct BlobKey(usize);
// or
enum BlobKey(usize);
```

Reply:
this particular code comes from generated part by tower-grpc. but other place can be fixed - in future PR